smap/sources/safarScraper.py

```diff
@@ -6,6 +6,7 @@
 # used for logging
 import logging
 import sys, traceback
+import simplejson as json
```

Comment:
json has an unicode bug, this was major reason to shift to simplejson.
References: 
[http://stackoverflow.com/questions/712791/what-are-the-differences-between-json-and-simplejson-python-modules](http://stackoverflow.com/questions/712791/what-are-the-differences-between-json-and-simplejson-python-modules)